sdk/core/Azure.Core.Experimental/src/Auth/Interfaces/IAuthorizationCodeFlowToken.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace System.ClientModel;
+
+/// <summary>
+/// Represents a provider that can provide a token.
+/// </summary>
+public interface IAuthorizationCodeFlowToken : IScopedFlowToken
+{
+    /// <summary>
+    /// Gets the authorization endpoint URI.
```

Comment:
Not important for the beta, but it would be really helpful to have a bit more context in these comments across interfaces, eventually.   I don't have deep identity knowledge and am not sure what I'd use them for.